src/pages/Home/Home.test.jsx

```diff
@@ -15,3 +17,11 @@ it("should render the text on the display", () => {
 
   expect(container).toMatchSnapshot();
 });
+
+it("should render the list view page when a user clicks list view", () => {
+  customRender(<Home />);
+  const { container } = customRender(<HomeListView />);
+  const listViewLink = screen.getByText("List View");
+  userEvent.click(listViewLink);
+  expect(container).toBeInTheDocument();
```

Comment:
isn't `container` always going to be in the document? might need a different way of testing this..